docs/laravel.md

```diff
@@ -73,3 +73,100 @@ The `octane:start` command can take the following options:
 * `--log-level`: Log messages at or above the specified log level
 
 Learn more about [Laravel Octane in its official documentation](https://laravel.com/docs/octane).
+
+## Laravel Apps As Standalone Binaries
+
+Using [FrankenPHP's application embedding feature](embed.md), it's possible to distribute Laravel
+apps as standalone binaries.
+
+Follow these steps to package your Laravel app as a standalone binary for Linux:
+
+1. Create a file named `static-build.Dockerfile` in the repository of your app:
+
+    ```dockerfile
+    FROM --platform=linux/amd64 dunglas/frankenphp:static-builder
+
+    # Copy your app
+    WORKDIR /go/src/app/dist/app
+    COPY . .
+
+    # Remove the tests and other unneeded files to save space
+    # Alternatively, add these files to a .dockerignore file
+    RUN rm -Rf tests/
+
+    # Copy .env file
+    RUN cp .env.example .env
+    # Change APP_ENV and APP_DEBUG to be production ready
+    RUN sed -i'' -e 's/^APP_ENV=.*/APP_ENV=production/' -e 's/^APP_DEBUG=.*/APP_DEBUG=false/' .env
+
+    # Make other changes to your .env file if needed
+
+    # Install the dependencies
+    RUN composer install --ignore-platform-reqs --no-dev -a
+
+    # Build the static binary
+    WORKDIR /go/src/app/
+    RUN EMBED=dist/app/ ./build-static.sh
+    ```
+
+    > [!CAUTION]
+    >
+    > Some `.dockerignore` files
+    > will ignore the `vendor/` directory and `.env` files. Be sure to adjust or remove the `.dockerignore` file before the build.
+
+2. Build:
+
+    ```console
+    docker build -t static-laravel-app -f static-build.Dockerfile .
+    ```
+
+3. Extract the binary:
+
+    ```console
+    docker cp $(docker create --name static-laravel-app-tmp static-laravel-app):/go/src/app/dist/frankenphp-linux-x86_64 frankenphp ; docker rm static-laravel-app-tmp
+    ```
+
+4. Populate caches:
+
+    ```console
+    ./frankenphp php-cli artisan optimize
+    ```
+
+5. Run database migrations (if any):
+
+    ```console
+    ./frankenphp php-cli artisan migrate
+    ````
+
+6. Generate app's secret key:
+
+    ```console
+    ./frankenphp php-cli artisan key:generate
+    ```
+
+5. Start the server:
+
+    ```console
+    ./frankenphp php-server
+    ```
+
+Your app is now ready!
+
+Learn more about the options available and how to build binaries for other OSes in the [applications embedding](embed.md)
+documentation.
+
+### Running Octane With Standalone Binaries
+
+It's even possible to package Laravel Octane apps as standalone binaries!
+
+To do so, [install Octane properly](#laravel-octane) and follow the steps described in [the previous section](#laravel-apps-as-standalone-binaries).
+
+Then, to start FrankenPHP in worker mode through Octane, run:
+
+```console
+PATH="$PWD:$PATH" ./frankenphp php-cli artisan octane:frankenphp
+```
+
+> ![CAUTION]
+>
+> For the command to work, the standalone binary **must** be named `frankenphp`.
```

Comment:
Just curious, but why?

Reply:
Because the Octane command looks for a binary named `frankenphp` in the path.

Reply:
Explained in efb13fc.